### PR TITLE
Update to Vanilla v1.8

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- Updated version of site to v1.8
- Removed margins from p.content and p.tutorial cards
- Updated `h3` to use correct markup

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8015/](http://0.0.0.0:8015/)
4. Compare with [live](https://developer.ubuntu.com/) and ensure that the site looks the same (or better)

### CODE REVIEW

File: templates/includes/base/header.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
...4 1.513-.335 2.592-.52 1.077-.18 2.316-.273 3.72-.273 1.262 0 2.307.178 3.14.533.835.356 1.5.853 1.998 1.494.498.64.85 1.408 1.053 2.3.202.898.304 1.882.304 2.96v8.904H60.74V16.34c0-.977-.066-1.81-.197-2.5-.132-.694-.35-1.25-.656-1.677-.303-.43-.712-.738-1.22-.93-.507-.197-1.14-.292-1.89-.292-.305 0-.62.013-.943.033-.326.02-.637.046-.93.076-.3.03-.56.066-.795.106-.236.04-.403.072-.504.09v13.387H50.77V9.236zM70.132 8.778h6.008v2.378h-6.008v7.32c0 .79.062 1.45.185 1.965.12.52.304.928.548 1.222.244.292.55.5.914.625.366.122.793.182 1.28.182.873 0 1.564-.097 2.075-.29.507-.194.853-.33 1.035-.413l.61 2.32c-.284.142-.788.324-1.51.547-.72.225-1.538.338-2.453.338-1.078 0-1.968-.137-2.67-.413-.7-.276-1.264-.686-1.69-1.235-.428-.548-.728-1.224-.9-2.027-.174-.805-.26-1.734-.26-2.79V4.356l2.834-.49V8.78h.002zM91.57 24.177c-.653.162-1.51.337-2.58.518-1.066.184-2.3.276-3.7.276-1.222 0-2.247-.178-3.08-.534-.837-.355-1.507-.858-2.014-1.51-.51-.65-.873-1.418-1.1-2.3-.22-.887-.335-1.866-.335-2.945V8.78h2.84v8.293c0 1.933.304 3.316.913 4.148.61.834 1.637 1.25 3.08 1.25.305 0 .62-.01.946-.032.322-.02.628-.045.913-.075.284-.03.546-.06.777-.09.234-.033.403-.066.505-.11V8.78h2.837v15.4h-.002z"/></g></svg>
```
Comment (anthonydillon) [inline templates/includes/base/header.html]: This image is missing `p-navigation__image`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
One small request

**Review by anthonydillon — COMMENTED** [review]
The mobile sidebar cta is menu could do with replacing `u-no-margin--top` with `u-no-margin--bottom`

File: templates/includes/base/header.html (hunk 1 above)
Comment (kwm14) [reply templates/includes/base/header.html]: @anthonydillon good spot. Have fixed up 👍

**Comment by kwm14** [comment]
@anthonydillon updated mobile sidebar cta and looks much better 💯 ready for another review 👍

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**